src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/JSON.txt

```diff
@@ -123,3 +123,25 @@ Error({Kind:ErrorKind.Div0})
 // Error records
 >> JSON(Filter([-2,-1,0,1,2], 1/Value>0))
 Error({Kind:ErrorKind.Div0})
+
+// Unwrapped tables
+>> JSON([1, 2, 3], JSONFormat.UnwrapValueTables)
+"[1,2,3]"
```

Comment:
You may want to add roundtrip test

Table(ParseJson(JSON([1,2,3])))

Reply:
The result will be a table of untyped values - which are currently not supported in the JSON function.